experimental/packages/api-logs/src/types/LogRecord.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Attributes } from '@opentelemetry/api';
+
+export interface LogRecord {
+   /**
+    * The time when the log record occurred as UNIX Epoch time in nanoseconds.
+    */
+   timestamp?: number;
+
+   /**
+    * Numerical value of the severity.
+    */
+   severityNumber?: number;
+
+   /**
+    * The severity text.
+    */
+   severityText?: string;
+
+   /**
+    * A value containing the body of the log record.
+    */
+   body?: string;
+
+   /**
+    * Attributes that define the log record.
+    */
+   attributes?: Attributes;
```

Comment:
Same `nested` comment as above -- perhaps add a "todo" somewhere?

The `best` approach will be for `Attributes` to support nesting (as per the OTLP protocol), but that will probably have other unintended consequences because it would mean that the SDK would start supporting nesting on trace attributes.